packetbeat/beater/packetbeat.go

```diff
@@ -49,11 +49,6 @@ type CmdLineArgs struct {
 
 var cmdLineArgs CmdLineArgs
 
-const (
-	defaultQueueSize     = 2048
```

Comment:
as queue sizes are configurable and both depend on shipper config it should be ok.